src/applications/personalization/profile/components/proof-of-veteran-status/ProofOfVeteranStatus.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import MobileAppCallout from '@department-of-veterans-affairs/platform-site-wide/MobileAppCallout';
+import ProofOfVeteranStatusCardExample from './proof-of-veteran-status-card-example';
+
+const ProofOfVeteranStatus = () => {
+  return (
+    <div id="proof-of-veteran-status">
+      <h2 className="vads-u-font-size--h3 vads-u-margin-top--4 vads-u-margin-bottom--1p5">
+        Proof of Veteran status
+      </h2>
+      <p>
+        You can download your military information to prove you served in the
+        United Status Uniformed Services. This status doesn’t entitle you to any
+        VA benefits. It can also help provide validation when trying to access
+        military discounts provided by retailers, restaurants and other
+        services. There are no forms to fill out. Your veteran status is
+        automatically generated by our system. If you feel your information is
+        incorrect, please visit your{' '}
+        <a href="/profile/personal-information">profile page.</a>
+      </p>
+      <div className="vads-u-margin--1 vads-u-margin-y--2">
+        <ProofOfVeteranStatusCardExample />
+      </div>
+      <div className="vads-u-font-size--h3">
+        <i className="fa fa-download" aria-hidden="true" />{' '}
```

Comment:
This section (all the icons, etc.) can be replaced with ```
<va-link
          role="link"
          download
          filetype="PDF"
          href={null}
          pages={1}
          text="Download Veteran status card"
                    onClick={() => {
            // eslint-disable-next-line no-console
            console.log('download PDF');
          }}
 />
 ``` 
 - it's got built in icons and things we need for the link.